src/bitsea/Sat/interpolator.py

```diff
@@ -103,18 +106,19 @@ def argument():
 from bitsea.Sat import interp2d
 from bitsea.Sat import SatManager as Sat
 from bitsea.utilities.mpi_serial_interface import get_mpi_communicator
+from sys import exit
 
 def interpolator(*,
 inmesh : str,
 serial : bool,
-maskfile : str,
-inputdir: str,
+maskfile : Path,
+inputfiles: List[str],
```

Comment:
`inputfiles` is a list of strings or a lists of `Paths`?

Reply:
I agree